targets/TARGET_STM/qspi_api.c

```diff
@@ -128,11 +266,94 @@ void qspi_prepare_command(const qspi_command_t *command, QSPI_CommandTypeDef *st
     }
 
     st_command->NbData = 0;
+    debug_if(qspi_api_c_debug, "qspi_prepare_command Out: InstructionMode %x Instruction %x AddressMode %x AddressSize %x Address %x DataMode %x\n",
+        st_command->InstructionMode, st_command->Instruction, st_command->AddressMode, st_command->AddressSize, st_command->Address, st_command->DataMode);
 }
+#endif /* OCTOSPI */
+
+
+#if defined(OCTOSPI1)
+qspi_status_t qspi_init(qspi_t *obj, PinName io0, PinName io1, PinName io2, PinName io3, PinName sclk, PinName ssel, uint32_t hz, uint8_t mode)
+{
+    OSPIM_CfgTypeDef OSPIM_Cfg_Struct = {0};
+    debug_if(qspi_api_c_debug, "qspi_init mode %u\n", mode);
 
+    __HAL_RCC_OSPI2_CLK_ENABLE();
```

Comment:
OSPI2 clock is used there. Maybe we could add some dependency on PeriphPin table ?

Reply:
done and reviewed to be generic. Made changes to other families too.